setup.py

```diff
@@ -47,6 +47,7 @@
     'Flask-CeleryExt>=0.1.0',
     'Flask-CLI>=0.2.1',
     'Flask-Mail>=0.9.1',
+    'fake-factory>=0.5.3',
```

Comment:
I would prefer no automatic name generation - force developers to write tests with explicit values.